README.md

```diff
@@ -6,7 +6,22 @@ alp is Access Log Profiler
 
 # Installation
 
-Download from https://github.com/tkuchiki/alp/releases
+### Binary distribution
+
+You can just pick your download [here](https://github.com/tkuchiki/alp/releases), put it in your PATH as alp, and set it as executable, e.g.
```

Comment:
Grammarly says `Consider removing it`, so remove it :)

```suggestion
You can pick your download [here](https://github.com/tkuchiki/alp/releases), put it in your PATH as alp, and set it as executable, e.g.
```

Reply:
Both sentences are grammatically correct, but I'm okay with that!